doc/src/manual/performance-tips.md

```diff
@@ -48,79 +54,79 @@ so all the performance issues discussed previously apply.
 
 ## Measure performance with [`@time`](@ref) and pay attention to memory allocation
 
-A useful tool for measuring performance is the [`@time`](@ref) macro. The following example
-illustrates good working style:
+A useful tool for measuring performance is the [`@time`](@ref) macro. We here repeat the example
+with the global variable above, but this time with the type annotation removed:
 
-```julia-repl
```

Comment:
Changed this from a 2-union type instability to global variables since unions are pretty fast now and more significantly do not allocate anymore, which was one of the point of this section.